std/algorithm/sorting.d

```diff
@@ -872,31 +876,53 @@ template multiSort(less...) //if (less.length > 1)
             alias ss = SwapStrategy.unstable;
             alias funs = less;
         }
-        alias lessFun = binaryFun!(funs[0]);
 
-        static if (funs.length > 1)
+        static if (funs.length == 1)
+            return sort!fun[0](r);
```

Comment:
I digged a bit deeper and it turns out that
1) this line was __never__ called in Phobos before (see the typo as a proof) -> I added a test to proof the fix!
2) multiSort is not recursive, it doesn't call itself like other templates do - this is just a specialization to sort which will be a bit faster. Hence I don't see any reason why we need to call `binaryFun` twice - we pass in just the alias in a lot of other functions.